apps/legacy-api/__tests__/controllers/StatsController.test.ts

```diff
@@ -35,7 +35,7 @@ describe('StatsController', () => {
         ),
         feeburn: expect.any(Number),
         auctionburn: expect.any(Number),
-        paybackburn: expect.any(String),
+        paybackburn: expect.any(Number) ?? expect.any(Array),
```

Comment:
lastly thinking to not impl a strict type on `paybackburn` to cater dev <string[0>] and ocean <number>